apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -2167,6 +2169,15 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                                 }
                                             </div>
                                         </Form.Field>
+                                        { ((initialValues?.state !== State.REVOKED) &&
+                                            SinglePageApplicationTemplate.id === template.id) ?
+                                            (<Message info={ true }>
+                                                { config.ui.productName } does not issue a&nbsp;
```

Comment:
Let's add i18n strings.

Reply:
Translations added in commit c8b25a5 and replicated the changes for `a_commit` in https://github.com/wso2-enterprise/asgardeo-app-extensions/pull/284/files